sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/Microsoft.Azure.HDInsight.Job.csproj

```diff
@@ -1,23 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <!-- Please do not move/edit code below this line -->
-    <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-    <!-- Please do not move/edit code below this line -->
-
     <PropertyGroup>
         <PackageId>Microsoft.Azure.HDInsight.Job</PackageId>
```

Comment:
PackageId and AssemblyName properties are not necessary as they are redundant and can be removed from this project.